packages/connect-core/src/entities/App.ts

```diff
@@ -117,4 +99,57 @@ export default class App {
       organization: null,
     }
   }
+
+  contract(): Contract {
```

Comment:
Added two helpers to facilitate the interaction using Ethers Contract and Interface for the app.